File: src/Alimento.java
public class Alimento extends Producto {
    private String fechaCaducidad;
    public Alimento(String nombre, double precio, String fechaImportacion, int stock, String fechaCaducidad) {
        super(nombre, precio, fechaImportacion, stock);
        this.fechaCaducidad = fechaCaducidad;
    }

    public String obtenerDatosAlimento() {
        return String.format("%s, fecha de caducidad: %s", super.obtenerInformacion(), fechaCaducidad);
    }
}

File: src/Alimento.py
from Producto import Producto


class Alimento(Producto):
    def __init__(self, nombre, precio, fecha_importacion, stock, fecha_caducidad):
        super().__init__(nombre, precio, fecha_importacion, stock)
        self.fecha_caducidad = fecha_caducidad

    def obtener_datos_alimento(self):
        return f"{super().obtener_informacion()}, fecha de caducidad: {self.fecha_caducidad}"
